Changes.md

```diff
@@ -11,6 +11,11 @@ Improvements
   - Grouped display can be enabled by default in a startup file by using `Gaffer.Metadata.registerValue( GafferSceneUI.RenderPassEditor.Settings, "displayGrouped", "userDefault", IECore.BoolData( True ) )`.
   - Dragging cells selected from the "Name" column now provides a list of the selected render pass names, rather than their paths.
 
+Fixes
+-----
+
+- RenderPassEditor : Fixed issue causing empty groups to be visible even though all of their children were hidden when hiding disabled render passes.
```

Comment:
This is redundant, since the grouping functionality hasn't been released separately, so nobody has seen the empty groups.

Reply:
Cool, makes sense. Removed in 94805d1.